client/app/recordings/show/template.jade

```diff
@@ -1,9 +1,47 @@
 include ../../shared/components/edit-buttons
-
+-
+  window.formRendered = function(){
```

Comment:
So, there are a couple of problems with this code:

1. Javascript should live in javascript land (.js files). Some of this looks like styling, which should just be written in css. The organization input should be added to the recording form through the recording schema (forms are generated from their corresponding model's schema), which is in `shared/recordings/model.js`. (We actually need re-usable organization selector component for a couple of use cases, I'm planning to work on that if you want to wait for it). In any case, none of this should live in the template.

2. We shouldn't attach anything to the global scope.

3. A lot of this is very brittle. For example, you're calling `document.getElementsByTagName("input")` and asserting that the second element is the organization input. What if we added a search bar to the header two weeks from now? That shouldn't affect the recording edit page, right? But it would break this logic.

4. In Backbone data shouldn't be stored in the DOM (actually I would say that's a general rule, DOM is slow, memory is fast). Take a minute and read through some of the other code in the codebase. See if you can pattern this functionality more closely on what's already there.